src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -25,26 +25,38 @@
 			{{ t('spreed', 'Locking the conversation prevents anyone to post messages or start calls.') }}
 		</div>
 		<div>
-			<input id="moderation_settings_lock_conversation_checkbox"
-				aria-describedby="moderation_settings_lock_conversation_hint"
-				type="checkbox"
-				class="checkbox"
-				name="moderation_settings_lock_conversation_checkbox"
-				:checked="isReadOnly"
-				:disabled="isReadOnlyStateLoading"
-				@change="toggleReadOnly">
-			<label for="moderation_settings_lock_conversation_checkbox">{{ t('spreed', 'Lock conversation') }}</label>
+			<h3>{{ t('spreed', 'Limit writing to a conversation') }}</h3>
+			<p>
+				<Multiselect id="limit_writing"
+					v-model="writingConversations"
+					:options="writingConversationOptions"
+					:placeholder="t('spreed', 'Limit writing to conversations')"
+					label="label"
+					track-by="value"
```

Comment:
please also set ":disabled" while saving. I hope that property is available on this component.